### DISCUSSION TITLE
Rare language and list of supported languages

### DISCUSSION THREAD

**Discussion by RollingTL** [intlify/vue-i18n#1000] [Q&A]
Could somebody provide list of supported languages?

I tried English and Russian and it works. But when I try date function for KZ locale (Kazakhstan) the function returns date in English. 

I have filled an [issue](https://github.com/intlify/vue-i18n-next/issues/999), but @kazupon closed issue with link to documentation, which contains no reply to my question. I think he has problems with English language and didn't understand what I mean. 

Anybody help please. I used this lib for two years but now started to look for alternative due to communication problems.

**Comment by RollingTL** [comment]
<img width="1299" alt="Screen Shot 2022-05-04 at 02 11 32" src="https://user-images.githubusercontent.com/16714526/166558554-2fcf3a38-4ddf-4983-9ea9-73a06943e372.png">

On the above picture you can see than common languages work, but a rare one - here Kazakh - is not working.

**Comment by MrTanoshii** [comment]
You're getting mixed up in your code

You have it declared as `kz` in your `dateTimeFormats.ts` but you're using `kk` in your template

I've tried it on my end and it works if you use `kk`

### datetimeFormats
```
  kk: {
    short: {
      year: "numeric",
      month: "long",
      day: "numeric",
    },
  },
```

### template
```
{{ $d(new Date(), "short", "kk") }}
```

### Result
![Screenshot 2022-05-17 175340](https://user-images.githubusercontent.com/47116127/168827535-dbf850b6-e1fe-46c9-be37-27a4ea942d11.png)

### Sidenote
You were very rude.
I started using this library yesterday so I have nowhere near your 2 years experience but I solved your issue by looking at the docs kazupon linked to so you really have no excuses.

**Reply by RollingTL** [reply]
May be I was rude sorry, but for reason simple reason that I tried to explain the problem several times.

> I started using this library yesterday so I have nowhere near your 2 years experience but I solved your issue by looking at the docs kazupon linked to so you really have no excuses.
You did't solve the problem. In my sample here there is no 'kz'. And the problem was that Chrome is supplying dates for only several main locales.


First time created an issue - https://github.com/intlify/vue-i18n-next/issues/999
Than here.
Than again an issue - https://github.com/intlify/vue-i18n-next/issues/1002
Ann only last issue was understood and I get a hint to the problem solution.
The problem was a bug in Chrome - https://bugs.chromium.org/p/chromium/issues/detail?id=842524

Usually lib owner is easily grasp what is the problem after verbal description as he knows the subject very well. I think author knows very well about date bugs in Crome, but simply could't understan that that was causing the problem.

And I have another bug I know exists, but afraid to make an issue as it is complex and occurs during SSR, so almost impossible to make minimal reproduction.
